qdrant-landing/content/articles/binary-quantization-openai.md

```diff
@@ -177,6 +179,14 @@ for combination in dataset_combinations:
     print(acc)
 ```
 
+Here is a selected slice of these results, with `rescore=True`:
+
+|Method|Dimensionality|Test Dataset|Recall|Oversampling|
+|-|-|-|-|-|
+|**Recommended** OpenAI text-embedding-3-large|3072|[DBpedia 1M](https://huggingface.co/datasets/Qdrant/dbpedia-entities-openai3-text-embedding-3-large-3072-1M) | 0.9966|3x|
+|OpenAI text-embedding-3-small|1536|[DBpedia 100K](https://huggingface.co/datasets/Qdrant/dbpedia-entities-openai3-text-embedding-3-small-1536-100K)| 0.9847|3x|
+|OpenAI text-embedding-3-large|1536|[DBpedia 1M](https://huggingface.co/datasets/Qdrant/dbpedia-entities-openai3-text-embedding-3-large-1536-1M)| 0.9826|3x|
```

Comment:
Why do we mark just one of the entries as recommended? `0.9826` might also be acceptable in many cases.

Reply:
Fair point, I'll update it to mention that this has the highest MTEB score among those mentioned.